arrow/factory.py

```diff
@@ -167,7 +167,8 @@ def get(self, *args, **kwargs):
 
             # (None) -> now, @ utc.
             if arg is None:
-                return self.type.utcnow()
+                raise NameError
+                # return self.type.utcnow()
```

Comment:
A `ValueError` is more appropriate in this situation.

Reply:
I had my doubts, but you're completely right, it's been changed.